packages/gitbook/src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -62,6 +63,7 @@ export function SpaceLayout(props: {
                 spaceId={context.space.id}
                 visitorAuthClaims={visitorAuthClaims}
             >
+                {customization.announcement?.enabled ? <Announcement context={context} /> : null}
```

Comment:
I suggest so that we can handle the `null` returns in one place, in `Announcement`
```suggestion
                <Announcement context={context} />
```